src/components/MainNav/MainNav.css

```diff
@@ -1,51 +1,127 @@
 /**
  * Remember your why
- * */
+ *
```

Comment:
![image](https://user-images.githubusercontent.com/31398520/118377648-c7acbc00-b58b-11eb-9d76-fa72b3e24a84.png)
We still have the bad formatting in this file, I've attached a screenshot of why this is bad formatting, it isn't aligned as it might have been on the original author's computer so I don't think we should use this kind of formatting when pushing code

Reply:
That closing bracket lines up with the nav underneath it. It's not aligned with the media because it's not the media queries closing bracket

Reply:
This one is about the spacing between the colons and the values! I'll push this one up!